meta_crush_saga.sh

```diff
@@ -10,7 +10,7 @@ while :
 do
     epoch_ms=$(date +%s%3N)
 
-    g++ -std=c++1z main.cpp -DKEYBOARD_INPUT="$keypressed" -DEPOCH_MS="$epoch_ms" -o renderer
+    g++ -std=c++1z main.cpp -DKEYBOARD_INPUT="$keypressed" -DEPOCH_MS="$epoch_ms" -fconcepts -o renderer
```

Comment:
Speaking of -Wall. It might be good to enable here at one point.

Reply:
Yes, and replace 1z with the official 17 or experimental 2a.